mkdocs.yml

```diff
@@ -15,13 +15,19 @@ theme:
 
 nav:
     - Home: index.md
+    - Getting Started: getting-started.md
     - User Guide:
+        - Overview: user-guide/index.md
+        - Installation: user-guide/installation.md
         - Writing Your Docs: user-guide/writing-your-docs.md
-        - Styling Your Docs: user-guide/styling-your-docs.md
+        - Choosing Your Theme: user-guide/choosing-your-theme.md
+        - Customizing Your Theme: user-guide/customizing-your-theme.md
         - Configuration: user-guide/configuration.md
         - Deploying Your Docs: user-guide/deploying-your-docs.md
-        - Custom Themes: user-guide/custom-themes.md
-        - Plugins: user-guide/plugins.md
```

Comment:
Please don't break URLs, there are many of them in the wild.
Some options would be:
* Don't change filenames despite changing sections.
* Use "redirects" plugin.

Reply:
These are not `user-guide` material. They never should have been put in that directory. Therefore, they are being moved. Redirects could be an option.

Reply:
Redirects have been defined using the plugin. Thanks for the suggestion.